src/viewer/viewer.ts

```diff
@@ -1,6 +1,5 @@
-import path from 'path';
 import * as vscode from 'vscode';
-import * as utils from './utils';
+import * as utils from '../utils';
 
 export class ViewerCustomEditor implements vscode.CustomReadonlyEditorProvider {
```

Comment:
Shouldn't we name this `ViewerWebview`, just to be consistent with `GalleryWebview`? The term "editor" should not be used anymore because I find it confusing, even though we know that under the hood this webview is implemented as an editor, specifically, `CustomReadonlyEditorProvider`.